src/app/members/components/MemberFormBuilder.tsx

```diff
@@ -135,21 +135,21 @@ export const MemberFormBuilder = () => {
 
   const handleSubmission = async (values: UserBuilder) => {
     try {
-      // When we insert data then it  give status not the insert item
-
+      // there is nothing in response in case of insert data
       const result = await createUser(values);
-      if (result) {
+
+      if (!result) {
+        form.reset();
         toast({
-          title: "Something went wrong",
+          title: "User created successfully",
+          description: "You will receive message shortly",
         });
       }
-      form.reset();
+    } catch (error) {
       toast({
-        title: "User created successfully",
-        description: "You will receive message shortly",
+        title: "User already exist",
+        description: "Please use other phone number",
```

Comment:
How do we know it's a phone number error?

Reply:
Sir , i was worng , showing expecting error but now i have fixed and showing exect error that is form supabse